### ISSUE TITLE
IFTTT [v2.2.1] 

### ISSUE THREAD

**Issue by Dijker** [athombv/homey#1082]
**What is your Homey version (Settings → System)?**

1.0.1 / IFTTT v2.2.1 

**What did you try to achieve?**
Re-connect Homey to IFTTT

*Please include detailed steps how we can reproduce the issue*
Removed the IFTTT Beta V2.2.1  (what worked for some time ;-) )
Installed IFTTT
Settings - IFTTT 
Button Connect to IFTTT is grey

[RED]Connect to IFTTTIt appears that this user is not the owner of this Homey, only the owner can connect to IFTTT. [/RED]

**What was the result?**
Could not Reconnect 

**What did you expect as result?**
A button to Connect. 

**Any other remarks**
Removed Both IFTTT Apps and disconnected Homey From the IFTTT Site, 
Restarted Homey, and Reinstalled Regular IFTTT 2.2.1 
No Change, Still not seen as Owner by the App, 
I am the only user and mentioned as Owner at Settings-Users

**Comment by philsniff** [comment]
i have the same issue :-(

**Comment by RobinBol** [comment]
Just managed to reproduce, thanks for reporting, fix should installing automatically on your Homeys in a couple of hours (or reboot Homey to force update).

**Comment by Dijker** [comment]
pls re-open: [Backspace (12x)]
I updated tha app. 
I get the button on connecting but get an Error: 
{"errors":["Could not get authorization token from athom-api"]}

![image](https://cloud.githubusercontent.com/assets/12412734/20456209/943a38dc-ae71-11e6-8da8-affe411c718a.png)


Ok, I disconnected from the IFTTT site my Homey and  tried a second time. This time it worked as expected.

Thanks! Issue Closed!
